common/mb_structs.h

```diff
@@ -6,12 +6,12 @@
 typedef struct mb_state mb_state_t;
 struct mb_state{
     // raw sensor inputs
-    float tb_angles[3]; // DMP filtered angles, tb_angles[3] is heading
+    float tb_angles[3]; // DMP filtered angles, tb_angles[2] is heading
```

Comment:
i think this is a typo in the original code provided by peter, just change it so that it makes more sense;

Reply:
I think tb_angles[3] is yaw (heading) so original comment should be correct

Reply:
so tb_angles is an array with 3 elements, the index in c/c++ starts from  0, meaning the third element in c/c++ should be tb_angles[2].

Reply:
oops, yeah make sense. Been switching between Julia and this haha